DemoServer/Controllers/Demos/CompareExchange/IndexCompareExchange/IndexCompareExchangeController.cs

```diff
@@ -0,0 +1,78 @@
+﻿using System.Threading.Tasks;
+using DemoCommon.Models;
+using DemoServer.Utils.Cache;
+using DemoServer.Utils.Database;
+using DemoServer.Utils.UserId;
+using Microsoft.AspNetCore.Mvc;
+#region Usings
+using System.Linq;
+using System.Collections.Generic;
+using Raven.Client.Documents.Linq;
+using Raven.Client.Documents.Session;
+using Raven.Client.Documents.Indexes;
+#endregion
+
+namespace DemoServer.Controllers.Demos.CompareExchange.IndexCompareExchange
+{
+    public class IndexCompareExchangeController : DemoCodeController
+    {
+        public IndexCompareExchangeController(UserIdContainer userId, UserStoreCache userStoreCache, MediaStoreCache mediaStoreCache,
+            DatabaseSetup databaseSetup) : base(userId, userStoreCache, mediaStoreCache, databaseSetup)
+        {
+        }
+
+        #region Demo
+        #region Step_1
+        public class Products_byUnitsInStock : AbstractIndexCreationTask<Product, Products_byUnitsInStock.IndexEntry>
```

Comment:
is lowercased "by" intentionally lowercased?

Reply:
fixed to uppercase